include/sound/hda-mlink.h

```diff
@@ -62,6 +62,11 @@ struct mutex *hdac_bus_eml_get_mutex(struct hdac_bus *bus, bool alt, int elid);
 
 int hdac_bus_eml_enable_offload(struct hdac_bus *bus, bool alt, int elid, bool enable);
 
+void hdac_bus_eml_set_mic_privacy_mask(struct hdac_bus *bus, bool alt, int elid,
+				       unsigned long mask);
+bool hdac_bus_eml_check_mic_privacy(struct hdac_bus *bus, bool alt, int elid);
+bool hdac_bus_eml_get_mic_privacy(struct hdac_bus *bus, bool alt, int elid);
```

Comment:
What is the difference between `check` and `get`?

Reply:
Ok, I got it after reading all the commits. `Check` if it is a mic privacy irq and `get` the mic privacy status.

Reply:
Had to go all the way the register spec to get that, but then I did understand that too. A short comment somewhere would not hurt thou.

Reply:
OK, let me do some renaming and commenting.